server/index.js

```diff
@@ -51,6 +50,37 @@ app.get('/api/notes', (req, res, next) => {
     .catch(err => next(err));
 
 });
+
+app.delete('/api/recordings/:id', (req, res, next) => {
+  const { id } = req.params;
+  if (!id) {
+    throw new ClientError(400, 'bad request');
+  }
```

Comment:
`id` will be truthy, because otherwise this route would not be matched. You can add validation to make sure it's a number though